pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -432,13 +348,13 @@ func nodeMatchesAllTopologyTerms(pod *v1.Pod, topologyPairs topologyToMatchedTer
 }
 
 //  nodeMatchesAnyTopologyTerm checks whether "nodeInfo" matches
-//  topology of any "term" for the given "pod".
-func nodeMatchesAnyTopologyTerm(pod *v1.Pod, topologyPairs topologyToMatchedTermCount, nodeInfo *framework.NodeInfo, terms []v1.PodAffinityTerm) bool {
+//  topology of any of the pod's anti affinity terms.
+func nodeMatchesAnyAntiAffinityTopologies(nodeInfo *framework.NodeInfo, state *preFilterState) bool {
```

Comment:
```suggestion
func nodeMatchesAnyAntiAffinityTerm(nodeInfo *framework.NodeInfo, state *preFilterState) bool {
```

As well as the comment:

```
// nodeMatchesAnyAntiAffinityTerm checks ...
```

Reply:
done.